app/controllers/oh_admin/jobs_controller.rb

```diff
@@ -0,0 +1,17 @@
+class OhAdmin::JobsController < ApplicationController
+  before_action :admin_session_required
+  before_action :find_project
+  layout 'admin'
+  helper JobApiHelper
+
+  def index
+    @response = JSON.parse(ApiJob.new(@project.id, params[:page]).get)
```

Comment:
How about renaming the `ApiJob` class to `JobApi`. As per rails convention, get method is to get a single object, not a collection, can we change that to `fetch`/`where`. 

It would be great if we can move the `get` method to a class method.